### PR TITLE
Assignment4

### PR DESCRIPTION
**Pull request by andriumon**
Tasks 6 and 7 without any other files modified

### CODE REVIEW

File: Assignment4/andriumon-a180385/Task07.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# In[32]:
+
+
+# TO DO
+q2 = prepareQuery('''
+  SELECT ?subject ?subject2 WHERE {
+      ?subject rdf:type <http://somewhere#Person>.
+      ?subclass rdfs:subClassOf <http://somewhere#Person>.
+      ?subject2 rdf:type ?subclass.
+  }
```
Comment (dgarijo) [inline Assignment4/andriumon-a180385/Task07.py:74]: Not correct. You need 1 variable only. Hint: use a UNION query (you almost have it).

File: Assignment4/andriumon-a180385/Task07.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+      ?property rdf:type ?class.
+  }
+  ''',
+  initNs = { "rdfs": RDFS, "rdf": RDF}
+)
+# Visualize the results
+for r in g.query(q3):
+    print(r)
+
+
```
Comment (dgarijo) [inline Assignment4/andriumon-a180385/Task07.py:138]: Same as above.

Also, missing exercises in rdflib

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please fix the issues

File: Assignment4/andriumon-a180385/Task07.py (hunk 2 above)
Comment (andriumon) [reply Assignment4/andriumon-a180385/Task07.py:138]: I commited a new version with what you told me to do. Please tell me if it's ok now

File: Assignment4/andriumon-a180385/Task07.py (hunk 1 above)
Comment (dgarijo) [reply Assignment4/andriumon-a180385/Task07.py:74]: you need to add chained properties with * to retrieve the full taxonomy of subclasses. You almost almost have it

File: Assignment4/andriumon-a180385/Task07.py (hunk 2 above)
Comment (dgarijo) [reply Assignment4/andriumon-a180385/Task07.py:138]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: Assignment4/andriumon-a180385/Task07.py (hunk 1 above)
Comment (andriumon) [reply Assignment4/andriumon-a180385/Task07.py:74]: Done. I think I'm not missing anything else so it should be fine

**Review by dgarijo — APPROVED** [review]

**Pull request merged**